src/integTestFixtures/groovy/org/gradle/playframework/fixtures/multiversion/PlayCoverage.groovy

```diff
@@ -9,5 +9,5 @@ final class PlayCoverage {
     private PlayCoverage() {}
 
     public static final VersionNumber DEFAULT = VersionNumber.parse(DEFAULT_PLAY_VERSION)
-    public static final List<VersionNumber> ALL = [VersionNumber.parse('2.4.11'), VersionNumber.parse('2.5.18'), DEFAULT]
+    public static final List<VersionNumber> ALL = [VersionNumber.parse('2.4.11'), VersionNumber.parse('2.5.18'), DEFAULT, "2.7.0"]
```

Comment:
The problem you were seeing was that this expected a `VersionNumber` and you were passing a `String`